openverse_catalog/dags/maintenance/add_license_url.py

```diff
@@ -0,0 +1,179 @@
+"""
+# Add license URL
+
+Add license_url to all rows that have NULL in their meta_data fields.
+The license_url is constructed from the license and license_version fields.
+
+This is a maintenance DAG that should run once.
```

Comment:
Nit: a few tweaks to improve markdown rendering

```suggestion
Add `license_url` to all rows that have `NULL` in their `meta_data` fields.
The `license_url` is constructed from the `license` and `license_version` fields.

This is a maintenance DAG that should be run once.
```